script.js

```diff
@@ -0,0 +1,99 @@
+const display = document.getElementById("display-value");
+const buttons = document.querySelectorAll(".btn");
+
+let currentValue = "0";
+let firstValue = null;
```

Comment:
You could create an object to store some important values per domain:
You could have a SELECTORS and CALCULATOR STATE constants to improve the overall logic:

```suggestion
const CALCULATOR_STATE = {
    INITIAL_VALUE: "0",
    ERROR_MESSAGE: "Error"
};
```